website/assets/apps/megamap/helpers/GeoStyles.js

```diff
@@ -53,7 +53,12 @@ export default class GeoStyles {
         }),
         points: objectMerge(pointStyle, {
           layout: {
-            'icon-image': 'Rail',
+            'icon-image': {
+              stops: [
+                [0, 'Rail'],
+                [6, 'RailIcon'],
```

Comment:
I think this, and the other icons in this file, should start at `minDetailZoom`, to match the other behavior at that zoom level